Resolver/Elastos-DID-Resolver-Specification_en.md

```diff
@@ -408,3 +383,246 @@ Member optional, contains other information related to the error. The value is d
   }
 }
 ```
+
+## DID Query
+
+DID Query is a method for querying DID documents on the ID side chain. The query scope is all latest DID documents in non-deactivated state, the expired documents are in the scope.
+
+### Query Request
```

Comment:
What about the following things?
- pagination of results (ex: if we have 10000 matching records we would kill the server/apps)
- shouldn't we be able to return the did document content directly? Otherwise we will always have to do 1 + N requests to the did chain to get result content, if only did strings are returned.